document-store/src/main/java/org/hypertrace/core/documentstore/postgres/query/v1/parser/builder/PostgresSelectExpressionParserBuilder.java

```diff
@@ -0,0 +1,10 @@
+package org.hypertrace.core.documentstore.postgres.query.v1.parser.builder;
+
+import org.hypertrace.core.documentstore.expression.impl.RelationalExpression;
+import org.hypertrace.core.documentstore.postgres.query.v1.PostgresQueryParser;
+import org.hypertrace.core.documentstore.postgres.query.v1.vistors.PostgresSelectTypeExpressionVisitor;
+
+public interface PostgresSelectExpressionParserBuilder {
+  PostgresSelectTypeExpressionVisitor buildFor(
```

Comment:
1. Change `buildFor` to `build` (Is there a specific reason for choosing `buildFor`)?

2. Is it necessary for `PostgresQueryParser` to be included as part of the interface? Could it instead be a member of the implemented builder object?

Reply:
1. Sure. (No specific reason)
2. Yes, can be. Will update.